src/main/java/org/tarantool/MsgPackLite.java

```diff
@@ -72,13 +72,6 @@
 
     public void pack(Object item, OutputStream os) throws IOException {
         DataOutputStream out = new DataOutputStream(os);
-        if (item instanceof Callable) {
```

Comment:
In some cases, you could do more with this ability.
You could simple not use this opportunity.